test/UniswapV3Staker.unit.spec.ts

```diff
@@ -630,8 +633,16 @@ describe('UniswapV3Staker.unit', async () => {
       it('has gas cost', async () => {
         await snapshotGasCost(subject({ to: recipient }))
       })
-
-      it('transfers the right amount of the reward token')
+      it('updates the reward available for the staker', async () => {
+        const rewardsAccured = await staker.rewards(
+          rewardToken.address,
+          rewardClaimer.address
+        )
+        await subject({ to: recipient })
+        expect(
+          await staker.rewards(rewardToken.address, rewardClaimer.address)
+        ).to.be.gt(rewardsAccured)
```

Comment:
I used this pattern for timestamps which are a bit unpredictable, but you can probably just make sure it incremented by the exact amount here.

I know! You're probably not even finished with the test..lol but leaving comments anyway.

Reply:
Yeah I saw this one, on my list to change. Better safe than sorry :)

Reply:
I think we should have a test that calculates the accuracy of the reward calculation, this can be done in another PR.

Reply:
makes sense. There should probably be a blank line b/w tests for consistency. Originally thought this was a test for claimRewards, I get how this would be tough to calculate.